docs/rst/platforms/platforms.rst

```diff
@@ -12,10 +12,11 @@ Vulcanexus ROS 2 all-in-one tool set, is officially available in the platforms s
       - Architecture
       - OS
     * - :ref:`Galactic Gamble <notes_galactic_latest>` v1
-      - amd64
+      - x86_64
       - Ubuntu Focal (20.04)
     * - :ref:`Humble Hierro <notes_humble_latest>` v2
-      - amd64
+      - arm64
+      - x86_64
```

Comment:
You'd need to include `.. include:: ../exports/alias.include` at the top of the file, I cannot make that suggestion

```suggestion
      - x86_64 |br|
        arm64
```